src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/ccd/CcdClient.java

```diff
@@ -43,30 +44,35 @@ public void completeAwaitingDcnProcessing(
         String caseTypeId = service.toUpperCase() + "_ExceptionRecord";
         String eventId = COMPLETE_AWAITING_DCN_PROCESSING_EVENT_ID;
 
-        StartEventResponse startEventResponse =
-            startEvent(authenticator, jurisdiction, caseTypeId, exceptionRecordCcdId, eventId);
-
-        Event event = Event
-            .builder()
-            .summary("Complete payment DCN processing")
-            .id(startEventResponse.getEventId())
-            .build();
-
-        CaseDataContent caseDataContent = CaseDataContent.builder()
-            .data(ImmutableMap.of(AWAITING_DCN_PROCESSING_FIELD_NAME, "No"))
-            .event(event)
-            .ignoreWarning(true)
-            .caseReference(exceptionRecordCcdId)
-            .eventToken(startEventResponse.getToken())
-            .build();
-
-        submitEvent(authenticator, jurisdiction, caseTypeId, exceptionRecordCcdId, caseDataContent);
-
-        log.info(
-            "Completed awaiting payment DCN processing. Exception record ID: {}, service {}",
-            exceptionRecordCcdId,
-            service
-        );
+        try {
+            StartEventResponse startEventResponse =
+                startEvent(authenticator, jurisdiction, caseTypeId, exceptionRecordCcdId, eventId);
+
+            Event event = Event
+                .builder()
+                .summary("Complete payment DCN processing")
+                .id(startEventResponse.getEventId())
+                .build();
+
+            CaseDataContent caseDataContent = CaseDataContent.builder()
+                .data(ImmutableMap.of(AWAITING_DCN_PROCESSING_FIELD_NAME, "No"))
+                .event(event)
+                .ignoreWarning(true)
+                .caseReference(exceptionRecordCcdId)
+                .eventToken(startEventResponse.getToken())
+                .build();
+
+            submitEvent(authenticator, jurisdiction, caseTypeId, exceptionRecordCcdId, caseDataContent);
+
+            log.info(
+                "Completed awaiting payment DCN processing. Exception record ID: {}, service {}",
```

Comment:
Is this text correct? shouldn't it be "Completed payment DCN processing"?

Reply:
Updated